app/MenuTitle.java

```diff
@@ -35,4 +35,14 @@ public static final void displayStoreName(String name) {
                 " ");
         MenuTitle.displayBorder();
     }
+
+    public static final void displayOptions(String[] optionsList) {
+        int range = optionsList.length;
```

Comment:
nice idea,, tho i think optionList.length nalang para sa for loop and hindi na range since one time use lang rin naman yung variable. andd I also think na it's more readable that way? since we'd know na we're looping the entire length of the optionsList. wdytt po

Reply:
tru tru